core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/aggregator/StdDevExpression.kt

```diff
@@ -0,0 +1,22 @@
+package ch.ergon.dope.resolvable.expression.unaliased.aggregator
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.expression.unaliased.aggregator.AggregateQuantifier.ALL
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.operator.FunctionOperator
+import ch.ergon.dope.validtype.NumberType
+
+class StdDevExpression(
```

Comment:
Would name it _StandardDeviationExpression_

Reply:
done